tests/Equinox.EventStore.Integration/StoreIntegration.fs

```diff
@@ -365,7 +367,53 @@ type Tests(testOutputHelper) =
         let! _ = service.Read cartId
         test <@ singleBatchBackwards @ batchBackwardsAndAppend @ singleBatchBackwards = capture.ExternalCalls @>
     }
-#endif
+    #else
+    [<AutoData(SkipIfRequestedViaEnvironmentVariable="EQUINOX_INTEGRATION_SKIP_EVENTSTORE")>]
+    let ``Can roundtrip against Store, correctly snapshotting to avoid redundant reads`` (ctx, skuId) = Async.RunSynchronously <| async {
```

Comment:
I duplicated the compaction tests as it was too confusing with ifdefs.

It's different functionality and the times at which we snapshot will be different because there'll be a different number of events in the stream

Reply:
Makes sense